Sources/ConcurrencyExtras/AnyHashableSendable.swift

```diff
@@ -40,3 +40,9 @@ extension AnyHashableSendable: CustomStringConvertible {
     String(describing: base)
   }
 }
+
+extension AnyHashableSendable: _HasCustomAnyHashableRepresentation {
+  public func _toCustomAnyHashable() -> AnyHashable? {
+    base as? AnyHashable
+  }
```

Comment:
Now the one behavior change to consider here is there's no way to recover the sendability anymore:

```swift
let x: AnyHashableSendable
let y: AnyHashable = x
y.base as? AnyHashableSendable      // `nil` now, previously not
y.base as? any Hashable & Sendable  // 🛑
```

> 🛑 Marker protocol 'Sendable' cannot be used in a conditional cast